sources/packages/api/src/constants/error-code.constants.ts

```diff
@@ -53,3 +53,11 @@ export const EDUCATION_PROGRAM_NOT_FOUND = "EDUCATION_PROGRAM_NOT_FOUND";
 
 export const STUDENT_ACCOUNT_APPLICATION_NOT_FOUND =
   "STUDENT_ACCOUNT_APPLICATION_NOT_FOUND";
+/**
+ * The user is already present on DB, either because the same BCeID user is
+ * trying to create an account application when it was already denied on the
```

Comment:
Please change from "denied on the past" to "denied in the past".

Reply:
Changed.